lib/matplotlib/figure.py

```diff
@@ -681,7 +681,7 @@ def add_subplot(self, *args, **kwargs):
             present figure but not in the figure's list of Axes.
 
         projection : {None, 'aitoff', 'hammer', 'lambert', 'mollweide', \
-'polar', 'rectilinear', str}, optional
+        'polar', 'rectilinear', str}, optional
```

Comment:
unrelated change

Reply:
My IDE did not like the way that this was wrapping lines. Fixed